api/alerts.go

```diff
@@ -142,28 +158,16 @@ func (a *Alerts) Add(viewName string, newAlert *Alert) (*Alert, error) {
 		"enabled":            graphql.Boolean(newAlert.Enabled),
 		"actions":            actions,
 		"labels":             labels,
+		"runAsUserId":        runAsUserIdPtr,
+		"queryOwnershipType": queryOwnershipType,
 	}
```

Comment:
```suggestion
	}
	
	if queryOwnershipType != "" {
	     variables["queryOwnershipType"] = queryOwnershipType
	 }
```

Reply:
```suggestion
		"queryOwnershipType": (*humiographql.QueryOwnershipType)(nil),
	}
	
	if queryOwnershipType != "" {
	     variables["queryOwnershipType"] = queryOwnershipType
	 }
```

Reply:
* `nil` => `panic: runtime error: invalid memory address or nil pointer dereference`
* `(*humiographql.QueryOwnershipType)(nil)` works for the construction of the mutation, but the graphql server is not happy about setting a custom scalar/enum to null. 
=>
`Error creating alert: Message: Argument 'input' has invalid value: Expected a value from enum com.humio.organizations.QueryHandle$QueryOwnership$QueryOwnershipType$@20a02e80, got: null (line 1, column 282):`

So the `(*graphql.String)(nil)` works because "null" is valid for a string. However, null for a scalar is not okay... the resulting mutation request:

```
..."labels":["some-label-1","some-label-2"],"queryOwnershipType":null,"queryStart":"3h","queryString":"some query","runAsUserId":null,"throttleField":"3h","throttleTimeMillis":7200000,"viewName":"DemoView"}}```

The solution would be to create two different struct tags, for each scenario, but that would result in some messy if-else statements. Thoughts?

Reply:
@danielamkaer Have found a "fix" and I'll update this PR later today.